SDL/SDLMain.cpp

```diff
@@ -546,6 +547,8 @@ float System_GetPropertyFloat(SystemProperty prop) {
 
 bool System_GetPropertyBool(SystemProperty prop) {
 	switch (prop) {
+	case SYSPROP_CAN_SHOW_FILE:
+		return true;
```

Comment:
Well, this should ideally have some ifdefs, not all SDLs support - such as BSDs.

-[Unknown]